src/components/SignUp.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import InputForm from "./Form";
+
+function SignUp() {
+  return(
+    <div>
+      <div className="flex flex-col items-center justify-center py-4 pt-12">
+        <p className="text-5xl font-light">Sign Up</p>
+        <p className="text-[#5cb85c] pt-2 text-lg my-2">Have an account?</p>
+      </div>
+
+      <div className="flex items-center justify-center px-4">
+        <div className="flex flex-col w-[640px] gap-y-5">
+          <InputForm fields={["Username","Email","Password"]} />
```

Comment:
The Input form should only make 1 input box each. So you can use InputForm 3 times here with a different field for each.

Reply:
Addressed. `InputForm` has been updated to show one field at a time. Thanks!